### PR TITLE
xmoto: update to 0.6.1.

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/xmoto/template [hunk 1]
```diff
@@ -13,27 +13,9 @@ short_desc="2D motocross platform game"
...
-pre_configure() {
-	# remove old ode version
-	rm -r src/ode
-	mv configure.{in,ac}
-	libtoolize
-	./bootstrap
-}
+distfiles="https://github.com/xmoto/xmoto/archive/${version}/${version}.tar.gz"
+checksum=209c8c38b1742d0620d40f90365c7a56f67c86da826c80a76d37fa46ee9c9b66
+nocross="Packing xmoto.bin; /bin/sh: xmoto: command not found"
```
Comment (ericonr) [inline srcpkgs/xmoto/template:18]: Did you try anything with qemu?

**Comment by sgn** [comment]
This should work without patching for musl:
```
LDFLAGS+=" -lintl"
```

**Comment by sgn** [comment]
I've spent sometime to read their `CMakeLists.txt`
They have `find_package(Intl, REQUIRED)` already, so the template can be a bit simpler:
```diff
diff --git a/srcpkgs/xmoto/template b/srcpkgs/xmoto/template
index d91f92f782..ae400bdce0 100644
--- a/srcpkgs/xmoto/template
+++ b/srcpkgs/xmoto/template
@@ -9,7 +9,7 @@ hostmakedepends="automake libtool gettext-devel intltool desktop-file-utils"
 makedepends="sqlite-devel zlib-devel libjpeg-turbo-devel libpng-devel
  libxml2-devel libxdg-basedir-devel glu-devel SDL-devel SDL_mixer-devel
  SDL_net-devel SDL_ttf-devel lua52-devel libcurl-devel libode-devel bzip2-devel
- gettext-devel"
+ gettext-libs"
 depends="${pkgname}-data"
 short_desc="2D motocross platform game"
 maintainer="Enno Boland <gottox@voidlinux.org>"
@@ -18,10 +18,6 @@ homepage="http://xmoto.tuxfamily.org/"
 distfiles="https://github.com/xmoto/xmoto/archive/${version}/${version}.tar.gz"
 checksum=209c8c38b1742d0620d40f90365c7a56f67c86da826c80a76d37fa46ee9c9b66
 
-if [ $XBPS_TARGET_LIBC = "musl" ]; then
-	export LDFLAGS+=" -lintl"
-fi
-
 pre_install() {
 	vtargetrun build/src/xmoto --pack build/bin/xmoto.bin bin/
 }
```

**Comment by ericonr** [comment]
Is libs enough for cross to work?

**Comment by sgn** [comment]
> Is libs enough for cross to work?

Yes, they declare the variable themselves https://github.com/xmoto/xmoto/blob/ebf4d1c32df8c5ab9dc52b2965a58aad23cb0c99/src/common/Locales.cpp#L72

And I tried compiling anyway ;)

**Pull request merged**